src/odemis/driver/smartpod.py

```diff
@@ -0,0 +1,835 @@
+# -*- coding: utf-8 -*-
+'''
+Created on 17 April 2019
+
+@author: Anders Muskens
+
+Copyright © 2012-2019 Anders Muskens, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+'''
+from __future__ import division
+from concurrent.futures import CancelledError, TimeoutError
+
+import os
+import logging
+import time
+from ctypes import *
+import copy
+import threading
+
+from odemis import model
+from odemis.util import driver
+from odemis.model import HwError, CancellableFuture, CancellableThreadPoolExecutor, isasync
+from operator import pos
+
+
+def add_coord(pos1, pos2):
```

Comment:
This code works well for what MoveRel() does, but it has limitations, which should be documented (before someone copies it and gets confused).
In particular, if pos1 and pos2 don't contain the same entries, it can do unexpected things. For example `{"x": 0, "y": 0}` + `{"x": 1, "z": 1}. What happens? y is dropped, and then the function fails because pos1 doesn't contain z.

At least indicate that: all axes in pos2 should be also in pos1.
The return value *only* contains the axes defined in pos2.